test/run.js

```diff
@@ -0,0 +1,20 @@
+const spawn = require('child_process').spawn
+
+const {start, stop} = require('./server')
+
+start(() => {
```

Comment:
You're right it doesn't cost much. On the other hand, I can't run tests only easily. If I want "all the checks", I can still do `npm run publish`.

This is a tiny detail and I'm okay with whatever you go with.